### PR TITLE
ION post

### PR DESCRIPTION
**Pull request by autonome**

### CODE REVIEW

File: content/post/099-identity-ipfs-ion.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+But identity is hard. There are no easy solutions for interoperable decentralized digital assertions of self. Privacy, security, validity, access control, surveillance, GDPR, KYC, anonymity. It's like Inception but each nested dream level is the same minefield, and those minefields are littered with projects that've tilted at the identity windmill, leaving naught behind but vinyl stickers slowly being covered up on our laptops.
+
+But hope has not forsaken this land. The [Decentralized Identity Foundation](https://identity.foundation/) has been plugging away at this challenge for many years, as have groups like [Rebooting the Web of Trust](https://www.weboftrust.info/). And there've been efforts to build decentralized identity systems on top IPFS, such as [IPID](https://github.com/johnnycrunch/ipid), the [Nomios.io](https://nomios.io/) and [the IPFS IDM](https://github.com/ipfs-shipyard/pm-idm), and most recently the [Ceramic Network](https://www.ceramic.network/).
```
Comment (momack2) [inline content/post/099-identity-ipfs-ion.md:14]: Just Nomios - no the

File: content/post/099-identity-ipfs-ion.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+![Ion architecture diagram](../img/099-identity-ipfs-ion/ion-architecture.png)
+
+Like HTTP, IPFS does not have user identity built into the protocol. However IPFS provides resiliency, validation and future-proofing features that HTTP cannot:
+
+* The content addressability of IPFS means that ION nodes who are pulling the CIDs off a blockchain or other underlying public network don't need to care *where* the transaction data resides - this means they can switch servers or datacenters or new storage nodes can come online without requiring any code or infrastructure changes and without the addresses needing to change.
+* ION nodes also don't need to worry about the data being manipulated or tampered with, because hash-based addressing means the cryptographic verification of the data is built into the network request itself.
+
+## IPFS in ION
+
+The ION implementation is in JavaScript (TypeScript) so it made sense for them to use js-ipfs as a node.js service. ION rolls up batches of identity transactions, publishes it through their IPFS node and then writes the address (CID) of that batch to the Bitcoin blockchain.
```
Comment (momack2) [inline content/post/099-identity-ipfs-ion.md:35]: Maybe “so the team gravitated to js-IPFS”? Don’t want to overstate that as the configuration we recommend for server side applications

File: content/post/099-identity-ipfs-ion.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+Hope has not forsaken this land. The [Decentralized Identity Foundation](https://identity.foundation/) has been plugging away at this challenge for many years, as have groups like [Rebooting the Web of Trust](https://www.weboftrust.info/). And there’ve been efforts to build decentralized identity systems on top of IPFS, such as [IPID](https://github.com/johnnycrunch/ipid), [Nomios.io](https://nomios.io/), [the IPFS IDM](https://github.com/ipfs-shipyard/pm-idm), and most recently the [Ceramic Network](https://www.ceramic.network/).
+
+However, today we’re celebrating the launch of a decentralized identity protocol and service from an unlikely place: Microsoft.
+
+Microsoft has been increasingly present in open-source tools and services in the past few years, and has now [launched a standards-based decentralized identity service called Ion](https://techcommunity.microsoft.com/t5/identity-standards-blog/ion-booting-up-the-network/ba-p/1441552).
```
Comment (renrutnnej) [inline content/post/099-identity-ipfs-ion.md:18]: Their announcement post calls it ION, but we refer to it as Ion in a few places, so I just want to make sure we're being consistent with the proper name.

**Comment by renrutnnej** [comment]
Very exciting! LGTM pending the Ion v ION clarification

File: content/post/099-identity-ipfs-ion.md (hunk 3 above)
Comment (autonome) [reply content/post/099-identity-ipfs-ion.md]: Yeah it's ION, thanks for catching the rogue Ions

File: content/post/099-identity-ipfs-ion.md (hunk 2 above)
Comment (autonome) [reply content/post/099-identity-ipfs-ion.md]: broader conversation that we can't solve here, but I don't want to reduce the importance of their choice nor that it works enough for them to feel confident shipping on it.

**Pull request merged**